src/Hng.Application/Features/Profiles/Handlers/UpdateProfileHandler.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Hng.Application.Features.Profiles.Handlers
 {
-    public class UpdateProfileHandler : IRequestHandler<UpdateProfileDto, Result<UpdateProfileResponseDto>>
+    public class UpdateProfileHandler : IRequestHandler<UpdateProfile, Result<UpdateProfileResponseDto>>
```

Comment:
Is this UpdateProfile class a command/request file? if yes, move to `Command` folder and name appropriately. check `Blog` feature folder.

Reply:
It is not a request nor command, it is just a record class that I used to move update profile properties rather than moving them one by one.

Reply:
ok